components/property_info.rst

```diff
@@ -0,0 +1,432 @@
+.. index::
+    single: PropertyInfo
+    single: Components; PropertyInfo
+
+The PropertyInfo Component
+==========================
+
+The PropertyInfo component provides the functionality to get information about class properties using metadata of
+popular sources.
+
+While the :doc:`PropertyAccess component</components/property_access/introduction>` allows you to read and write values
+to/from objects and arrays, the PropertyInfo component works solely with class definitions to provide information
+such as data type and visibility about properties within that class.
+
+Similar to PropertyAccess, the PropertyInfo component blurs the lines between class properties such as ``$property``
```

Comment:
Previous discussion on rewording of "blurs the lines between" here: https://github.com/symfony/symfony-docs/pull/5974#discussion_r46890491

Perhaps "combines both"?